lib/handle_callback.ts

```diff
@@ -80,14 +103,12 @@ export async function handleCallback(
     ...options?.cookieOptions,
   };
   setCookie(response.headers, cookie);
+  const session = await options?.sessionDataGetter?.(tokens.accessToken);
```

Comment:
Probably we should assert `session !== null` below this line?

Reply:
My reasoning is that perhaps there may be a use case that doesn't care about storing any session data. Rather, the dev just wants to know whether a session exists. Is this a silly use case? WDYT?

Reply:
The example in README now uses `await getSessionData(request) === null` as the check for session existence. If `sessionDataGetter` returns `null`, isn't it difficult to distinguish the 2 states?: `The user doesn't have session` and `The user have session with data === null`

Reply:
Ah, yes. That makes sense. I've added a guard below.